dist/cordova-plugin-iosrtc.js

```diff
@@ -2529,6 +2578,12 @@ function registerGlobals() {
 	navigator.webkitGetUserMedia            = getUserMedia;
 	navigator.mediaDevices.getUserMedia     = getUserMedia;
 	navigator.mediaDevices.enumerateDevices = enumerateDevices;
+
+	// Restore Callback support
+	if (!doNotRestoreCallbacksSupport) {
```

Comment:
@hthetiot should this be above `navigator.webkitGetUserMedia            = getUserMedia;`?
I'm facing an issue where it still throws an error while using `getUserMedia`